terraform/dev/terragrunt.hcl

```diff
@@ -43,10 +43,10 @@ generate "dev_tfvars" {
   fam_console_idp_name_bceid = "TEST-BCEIDBUSINESS"
   forest_client_api_base_url_test = "${local.common_vars.inputs.forest_client_api_test_base_url}"
   use_override_proxy_endpoints = true
-  dev_override_bcsc_userinfo_proxy_endpoint = "https://6mud7781pe.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/dev"
+  dev_override_bcsc_userinfo_proxy_endpoint = "https://xy7pk81p4h.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/dev"
```

Comment:
Change from previous prod API Gateway to dev API Gateway for easy testing in dev.